doc/cmake/BuildDocumentation.cmake

```diff
@@ -13,5 +13,5 @@ endif()
 execute_process( COMMAND ${XELATEX} -interaction=${THIS_TEX_INTERACTION} ${INNAME}.tex TIMEOUT 600 )
 execute_process( COMMAND ${XELATEX} -interaction=${THIS_TEX_INTERACTION} ${INNAME}.tex TIMEOUT 600 )
 execute_process( COMMAND ${XELATEX} -interaction=${THIS_TEX_INTERACTION} ${INNAME}.tex TIMEOUT 600 )
-file( COPY "${INNAME}.pdf" DESTINATION "${ORIGINAL_CMAKE_BINARY_DIR}/doc-pdf/" )
-file( RENAME "${ORIGINAL_CMAKE_BINARY_DIR}/doc-pdf/${INNAME}.pdf" "${ORIGINAL_CMAKE_BINARY_DIR}/doc-pdf/${OUTNAME}.pdf")
+file( COPY "${INNAME}.pdf" DESTINATION "${ORIGINAL_CMAKE_BINARY_DIR}/pdf/" )
```

Comment:
The CMake binary directories used are a little confusing here, not that you added the confusion, I think I did originally.  I wonder if this could be cleaned up to just use PROJECT directory variables instead.  Everything seems to work just fine now though so probably not anything for now.